libsolidity/interface/OptimiserSettings.h

```diff
@@ -41,10 +41,10 @@ struct OptimiserSettings
 
 			// should have good "compilability" property here.
 
-			"Tpeul"                    // Run functional expression inliner
+			"Tpeiul"                   // Run functional expression inliner
```

Comment:
Why run the full inliner? The comment only mentions the functional expression inliner.

Reply:
We'll have to run full inliner twice. One after UnusedFunctionParameterPruner and once after this step. The sequence might have to be modified